spec.go

```diff
@@ -223,6 +229,9 @@ func createLibcontainerMount(cwd string, m specs.Mount) *configs.Mount {
 			source = filepath.Join(cwd, m.Source)
 		}
 	}
+	if m.Type == "cgroup" && flags&syscall.MS_RDONLY == 0 {
```

Comment:
I think we can take this out. runc is low level enough for people to know what they are doing. Also, as we show ro in the recommendation, I think it should suffice.